eth_utils/address.py

```diff
@@ -16,7 +18,10 @@
 )
 
 
-def is_hex_address(value):
+Address = NewType('Address', str)
+
+
+def is_hex_address(value: str) -> bool:
```

Comment:
These `is_thing` functions should probably allow `Any` for their `value` parameter since it is completely valid to pass in *whatever*.